mlflow/java/scoring/src/main/java/org/mlflow/models/Model.java

```diff
@@ -82,6 +91,11 @@ public static Model fromConfigPath(String configPath) throws IOException {
     return Optional.ofNullable(this.runId);
   }
 
+    /** @return The MLflow model's uuid */
+  public Optional<String> getModelUuid() {
```

Comment:
nit: <4 Acronym naming convention in Java typically preserves the upper case acronym, particularly if the std library preserves. 
```java
public Optional<String> getModelUUID() {
```

Reply:
Good point!

Reply:
@BenWilson2 Can we use `getModelUuid` since we already have `getRunUuid`?

https://github.com/mlflow/mlflow/blob/c635d1aa12e1749ab1321128ac61c0f3e6309c1d/mlflow/server/js/src/experiment-tracking/reducers/Reducers.test.js#L166